examples/04_vector_field/01_3d_vector_field.py

```diff
@@ -0,0 +1,73 @@
+"""
+Generating a Random 3D Vector Field
+-----------------------------------
+
+In this example we are going to generate a random 3D vector field with a
+Gaussian covariance model. The mesh on which we generate the field will be
+externally defined and it will be generated by PyVista.
+"""
+# sphinx_gallery_thumbnail_path = 'pics/GS_3d_vector_field.png'
+import numpy as np
+import gstools as gs
+import pyvista as pv
+
+# mainly for setting a white background
+pv.set_plot_theme("document")
+
+###############################################################################
+# create a uniform grid with PyVista
+nx, ny, nz = 40, 30, 10
+dim, spacing, origin = (40, 30, 10), (1, 1, 1), (-10, 0, 0)
+mesh = pv.UniformGrid(dim, spacing, origin)
+x = mesh.points[:, 0]
+y = mesh.points[:, 1]
+z = mesh.points[:, 2]
+
+###############################################################################
+# create an incompressible random 3d velocity field on the given mesh
+model = gs.Gaussian(dim=3, var=3, len_scale=1.5)
+srf = gs.SRF(model, generator='VectorField', seed=198412031)
+srf((x, y, z), mesh_type='unstructured')
+
+# add a mean velocity in x-direction
+srf.field[0, :] += 0.5
```

Comment:
Couldn't this be provided by a mean? I think the new mean-normalizer-trend thingy is capable of dealing with vectors. But this is just fine here ;-)

Reply:
I'll update the example when the normalizer is ready. This example is gonna show a lot of fancy things ;-)